parameter/ParameterMgr.cpp

```diff
@@ -1820,6 +1822,37 @@ CParameterMgr::CCommandHandler::CommandStatus CParameterMgr::showMappingCommandP
     return CCommandHandler::ESucceeded;
 }
 
+CParameterMgr::CCommandHandler::CommandStatus CParameterMgr::listMappingsCommandProcess(
+    const IRemoteCommand &remoteCommand, string &strResult)
+{
+    CElementLocator elementLocator(getSystemClass(), false);
+
+    CElement *pLocatedElement = nullptr;
+
+    if (!elementLocator.locate(remoteCommand.getArgument(0), &pLocatedElement, strResult)) {
+        return CCommandHandler::EFailed;
+    }
+
+    if (!pLocatedElement) {
+        // List from root folder
+        pLocatedElement = getSystemClass();
+    }
+
+    vector<string> paths;
+    pLocatedElement->getSubpaths(paths);
+
+    for (string path : paths) {
+        string mapping;
+        if (getParameterMapping(path, mapping)) {
```

Comment:
What you seem to want isn't the terminal element (i.e. leaves, i.e. parameters) but the SubsystemObjects (the elements with an instantiation mapping).